[JS]: incompatible with typscript 4.7.4

We're currently using the ditto js sdk v2.4 with typescript 4.7.4.
Since 3.0 is out, we wanted to upgrade but our build fails with the following errors:

```
Error: node_modules/@eclipse-ditto/ditto-javascript-client-dom/dist/api/src/model/policies.model.d.ts:30:9 - error TS2611: 'id' is defined as a property in class 'EntityWithId', but is overridden here in 'Policy' as an accessor.
30     get id(): string;
           ~~
Error: node_modules/@eclipse-ditto/ditto-javascript-client-dom/dist/api/src/model/policies.model.d.ts:62:9 - error TS2611: 'id' is defined as a property in class 'EntityWithId', but is overridden here in 'Entry' as an accessor.
62     get id(): string;
           ~~
Error: node_modules/@eclipse-ditto/ditto-javascript-client-dom/dist/api/src/model/policies.model.d.ts:119:9 - error TS2611: 'id' is defined as a property in class 'EntityWithId', but is overridden here in 'Subject' as an accessor.
119     get id(): string;
            ~~
Error: node_modules/@eclipse-ditto/ditto-javascript-client-dom/dist/api/src/model/policies.model.d.ts:143:9 - error TS2611: 'id' is defined as a property in class 'EntityWithId', but is overridden here in 'Resource' as an accessor.
143     get id(): string;
            ~~
Error: node_modules/@eclipse-ditto/ditto-javascript-client-dom/dist/api/src/model/things.model.d.ts:39:9 - error TS2611: 'id' is defined as a property in class 'EntityWithId', but is overridden here in 'Thing' as an accessor.
39     get id(): string;
           ~~
Error: node_modules/@eclipse-ditto/ditto-javascript-client-dom/dist/api/src/model/things.model.d.ts:91:9 - error TS2611: 'id' is defined as a property in class 'EntityWithId', but is overridden here in 'Feature' as an accessor.
91     get id(): string;
```

Apparently you can only override fields when they are declared as an abstract field to begin with.

--- a/javascript/lib/api/src/model/model.ts
+++ b/javascript/lib/api/src/model/model.ts
@@ -68,7 +68,7 @@ export abstract class EntityModel {
  * Entity with an 'id' field
  */
 export abstract class EntityWithId extends EntityModel {
-  id!: string;
+  abstract id: string;
 
   equals(toCompare: EntityWithId): boolean {
     return super.equals(toCompare) && this.id === toCompare.id;

